change/@internal-react-components-4d499d50-2da5-463e-9080-800f5baaef5b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Initial changes for showing uploaded files in sendbox",
```

Comment:
I would make it more descriptive like
Add a prop `onRenderAttachedFiles` for rendering uploaded files in sendbox.

Reply:
Done